app/feature/feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/step/survey/TerminationSurveyDestination.kt

```diff
@@ -149,6 +165,40 @@ private fun TerminationSurveyScreen(
         navigateUp = navigateUp,
         closeTerminationFlow = closeTerminationFlow,
       ) {
+        AnimatedVisibility(uiState.showEmptyQuotesDialog) {
```

Comment:
I'd expect the dialog itself to not take any effect from the AnimatedVisibility around it really, since the dialog is not a normal UI but it's a new window on top of everything.
I think we can replace this with an `if statement` instead.